### PR TITLE
adjust ribbon style

### PR DESCRIPTION
**Pull request by osodebailar**
Only optical adjustment:

- ribbon default background color is now the material button color, because the ribbons have button functionality
- add ripple effect on touch, introduced by @Andries-Smit for date/time 
- add an icon to the left.

Merge if you like it.

ribbon_default_warning_dark
ribbon_default_warning_light
ribbon_warning_dark
ribbon_warning_light

### CODE REVIEW

**Comment by Andries-Smit** [comment]
@osodebailar why not make them actual buttons?

File: app/src/main/java/info/nightscout/androidaps/plugins/general/overview/OverviewFragment.kt [hunk 1]
```diff
@@ -832,6 +833,8 @@ class OverviewFragment : DaggerFragment(), View.OnClickListener, OnLongClickList
         binding.activeProfile.text = profileFunction.getProfileNameWithRemainingTime()
         binding.activeProfile.setBackgroundColor(profileBackgroundColor)
         binding.activeProfile.setTextColor(profileTextColor)
+        val drawableLeft: Array<Drawable?> = binding.activeProfile.compoundDrawables
+        if (drawableLeft[0] != null) profileTextColor.let { drawableLeft[0]!!.setTint(it) }
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/general/overview/OverviewFragment.kt:837]: drawableLeft[0]?.let { it.setTint(profileTextColor) }

Comment (MilosKozak) [reply app/src/main/java/info/nightscout/androidaps/plugins/general/overview/OverviewFragment.kt:837]: the same bellow

**Comment by MilosKozak** [comment]
we use different icon for profile everywhere

**Comment by osodebailar** [comment]
> 

Oh do Not See the other ones . Where I can find them ? 
and in the end , what icon would we take ?

**Comment by osodebailar** [comment]
> @osodebailar why not make them actual buttons?

Because until now they are not buttons . And to Convert them to buttons means to adjust the whole code for background and so on .
Same as in date/ time select .

**Comment by MilosKozak** [comment]
> > 
> 
> Oh do Not See the other ones . Where I can find them ? and in the end , what icon would we take ?

the same code is repeating multiple times

File: app/src/main/java/info/nightscout/androidaps/plugins/general/overview/OverviewFragment.kt (hunk 1 above)
Comment (osodebailar) [reply app/src/main/java/info/nightscout/androidaps/plugins/general/overview/OverviewFragment.kt:837]: Oh yes thx, mine was a little bit to complicated

**Comment by osodebailar** [comment]
> multiple

changed it to the settings profile icon
ribbon_profile_icon_dark



> > > 
> > 
> > 
> > Oh do Not See the other ones . Where I can find them ? and in the end , what icon would we take ?
> 
> the same code is repeating multiple times

simplified the code in overviewfragment

File: app/src/main/java/info/nightscout/androidaps/plugins/general/overview/OverviewFragment.kt (hunk 1 above)
Comment (osodebailar) [reply app/src/main/java/info/nightscout/androidaps/plugins/general/overview/OverviewFragment.kt]: simlified statement and former code

**Comment by osodebailar** [comment]
> we use different icon for profile everywhere

tokk the same as we have in settings for local profile

**Comment by osodebailar** [comment]
@MilosKozak so far ready from my side .

**Comment by swissalpine** [comment]
Can we hide the buttons for low resolution devices? Thanks

**Comment by osodebailar** [comment]
@swissalpine but if we hide them , you lost function

**Comment by Philoul** [comment]
@osodebailar Concerning profile, you put icon of local profile plugin but everywhere "profile" icon is the 5 pointed star (profile switch dialog, profile switch button in actions, automation, profile viewer, compare profile...)

we have 2 identical icons available:
`@drawable/ic_actions_profileswitch`
`@drawable/ic_home_profile`

I think this icon is better than local profile plugin icon for profile in overview (short click show profile viewer, long click profile switch, both dialogs has the same 5 pointed star in header)

**Comment by osodebailar** [comment]
Ok . But a 5 star icon is not very intuitive for a profile . The connection 5 star to Profil is hard to understand

**Comment by Philoul** [comment]
It's another question (should we change profile icon or not to have a better understanding?)
I think consistancy of the 2 icons (if accepted) is the most important here.

In my mind (but I don't know if it was the reason of this choice in the past) the 5 pointed star illustrate the 5 main items you adjust in a profile 🤔:
- Insulin
- IC
- ISF
- Basal
- Target

**Comment by osodebailar** [comment]
@Philoul  change the icon to five star 
@swissalpine remove the ribbons if style is set to low res

@MilosKozak : if you are satisfied you can merge

file:///home/developer/Pictures/ribbon_fivestar_light.PNG

**Comment by Philoul** [comment]
Why you created a new star icon (with a different shape) and a new target icon (not consistant with existing target icon used everywhere in aaps)?

If it's just the size, I can generate smaller icons with the same shape, but it's probably possible (and better) to use existing xml files and resize them by code or xml parameter in the layout...

when I saw that there were 2 drawing files for the same profile icon, my first idea was to merge them in one file in a dedicated PR to have only one drawing for one icon used everywhere and ensure consistancy...

**Comment by osodebailar** [comment]
@Philoul i need the other icon because it’s much more smaller .
If you want to create the right one , yes it’s up to you and i thank you If you can send me .

**Comment by Philoul** [comment]
Ok, I'll create the 2 icons with your size and will send it to you (mail or PR on your repo 😉)

**Comment by osodebailar** [comment]
@Philoul thx .
Mail is the best I think: 
andreas.eggen@gmail.von

**Comment by swissalpine** [comment]
@osodebailar Sorry, I edited the post a minute after writing: I didn't mean the ribbon, just the icon.

File: app/src/main/java/info/nightscout/androidaps/skins/SkinLowRes.kt [hunk 2]
```diff
@@ -70,6 +70,8 @@ class SkinLowRes @Inject constructor(private val config: Config) : SkinInterface
                 setMargins(0,0,0,0)
             }
             graphCard.layoutParams = paramGraph
+
+            loopLayout.visibility = GONE
```
[the remark this answers is not in the document]
Comment (osodebailar) [reply app/src/main/java/info/nightscout/androidaps/skins/SkinLowRes.kt:74]: I never see your comment .
@swissalpine you bring me to work …. 10 seconds of  coding for nothing 😂😂

**Comment by osodebailar** [comment]
@MilosKozak 
ready to mergenow. 

And many thx to @Philoul for the icon work.
ribbon_dark

File: app/src/main/java/info/nightscout/androidaps/skins/SkinLowRes.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (swissalpine) [reply app/src/main/java/info/nightscout/androidaps/skins/SkinLowRes.kt]: I'm so sorry! ;-)

**Pull request merged**